<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolbarWindow</class>
 <widget class="QMainWindow" name="ToolbarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>54</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/pics/PacmanTray-arch_logo</normaloff>:/pics/PacmanTray-arch_logo</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCheck_for_updates"/>
   <addaction name="actionUpdate_now"/>
   <addaction name="actionErrors"/>
  </widget>
  <action name="actionCheck_for_updates">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/pics/updates.png</normaloff>:/pics/updates.png</iconset>
   </property>
   <property name="text">
    <string>Check now</string>
   </property>
   <property name="toolTip">
    <string>Downloads and updates the pacman's database</string>
   </property>
  </action>
  <action name="actionUpdate_now">
   <property name="icon">
    <iconset resource="../QPacman/qpacman.qrc">
     <normaloff>:/pics/fullupdate.png</normaloff>:/pics/fullupdate.png</iconset>
   </property>
   <property name="text">
    <string>System Update</string>
   </property>
   <property name="toolTip">
    <string>Start updating of the system</string>
   </property>
  </action>
  <action name="actionErrors">
   <property name="icon">
    <iconset resource="../QPacman/qpacman.qrc">
     <normaloff>:/pics/kpk-important.png</normaloff>:/pics/kpk-important.png</iconset>
   </property>
   <property name="text">
    <string>Errors</string>
   </property>
   <property name="toolTip">
    <string>Shows the last errors</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../QPacman/qpacman.qrc"/>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
